src/terrain/services/permanent_id_requests.clj

```diff
@@ -184,46 +185,102 @@ If this dataset accompanies a paper, please contact us with the DOI for that pap
 (defn- move-folder
   [user {:keys [id path]} dest-path]
   (try+
-   (data-info-client/move-single user id dest-path)
-   (ft/path-join dest-path (ft/basename path))
-
+   (-> (data-info-client/move-single user id dest-path)
+       :body
+       (json/decode true)
+       :async-task-id)
    (catch Object e
      (log/error e)
      (email/send-permanent-id-request-data-move-error path dest-path current-user (:body e (str e)))
-     path)))
+     nil)))
```

Comment:
Now returns the async task ID for the move, rather than the move destination path.